packages/react-next/src/components/Coachmark/Coachmark.styles.ts

```diff
@@ -260,9 +258,12 @@ export function getStyles(props: ICoachmarkStyleProps): ICoachmarkStyles {
           },
         },
       },
+      !isMeasuring &&
```

Comment:
Why the change in logic here?

Reply:
I don't remember the precise details (this work was done weeks ago), but IIRC the issue was that the measurement was now being taken such that without this change, it would determine that the size of the container to render into was 0x0, making it effectively invisible.